README.rst

```diff
@@ -2,6 +2,31 @@
 ts_audio_broadcaster
 ####################
 
-``ts_audio_broadcaster`` is a package in the `LSST Science Pipelines <https://pipelines.lsst.io>`_.
+``ts_audio_broadcaster`` is is a component of LSST Telescope and Site software. It is used to generate web service that broadcasts audio from microphones installed on the observatory facilities.
```

Comment:
```suggestion
``ts_audio_broadcaster`` is a component of the Vera C. Rubin Telescope and Site software. It is used to generate web service that broadcasts audio from microphones installed on the observatory facilities.
```
You had a double is here and we are part of the Vera C. Rubin Telescope and Site software. I think some of the templates are still a little bit behind the times.